.github/workflows/build-rapids.yml

```diff
@@ -85,8 +85,10 @@ jobs:
           role-duration-seconds: 43200 # 12h
       - name: Run command # Do not change this step's name, it is checked in parse-job-times.py
         env:
-          CCCL_TAG: ${{ inputs.override_cccl_tag }}
-          CCCL_VERSION: ${{ inputs.override_cccl_version }}
+          # TODO: Temporarily hardcoding the override while RAPIDS gets updated
```

Comment:
Make sure we back this out at some point, otherwise the workflow_dispatch options won't work.

Reply:
The plan is to back it out when RAPIDS works on CCCL 3.2 compatibility. Unfortunately right now we're not in a position to test on main due to the large number of deprecations for 3.2 because we compile with warnings as errors. If it's too big a lift we could change our compilations to ignore the CCCL warnings until CCCL 4.0, but that would also be more work to do so we'll cross that bridge once 3.1 is working.